README.md

```diff
@@ -8,9 +8,42 @@ I want to build a time keeping app with a pdf invoicing feature. 
 
 * users should be able to move tasks up and down and all other crud functionality 
```

Comment:
This is fine, but make it into a formal README that the public (or an employer) might read with titles and formatting.